test/helpers.go

```diff
@@ -51,3 +53,24 @@ func InitConfigForIntegrationsTesting() {
 	// Load normal config from environment variables and config files
 	cmdutil.InitConfig()
 }
+
+func MockEventContainerV2(key string) eventsapi.EventContainer {
+	eventContainer := eventsapi.EventContainer{
+		EventVersion: eventsapi.EventVersion2,
+	}
+
+	eventV2 := eventsapi.EventV2{
```

Comment:
I'm not sure how much this is used by we might consider making a constructor inside `eventsapi` like so: 

```go
func NewEventContainer(event Event) *EventContainer {
	jsonEvent, _ := json.Marshal(event)

	return &EventContainer{
		EventVersion: event.Version(),
		EventData:    jsonEvent,
	}
}
```

if we're ever actually doing this in production code.

Reply:
As of right now this is only done here in the test helper, should I write the constructor anyways or leave as is?

Reply:
No worries for now.